spec/unit/dbconsole_spec.rb

```diff
@@ -0,0 +1,22 @@
+RSpec.describe 'dbconsole' do
```

Comment:
I don't know that this particular unit spec is necessary - it's essentially the same as [unit/run_spec.rb](https://github.com/technekes/nib/blob/master/spec/unit/run_spec.rb)